src/lib/viewers/media/DashViewer.js

```diff
@@ -492,7 +491,10 @@ class DashViewer extends VideoBaseViewer {
     resize() {
         let width = this.videoWidth || 0;
         let height = this.videoHeight || 0;
-        const viewport = this.wrapperEl.getBoundingClientRect();
+        const viewport = {
+            height: this.wrapperEl.clientHeight,
```

Comment:
Looks like windows chrome was giving slightly larger values for getBoundingClientRect. I'll verify this with IE/Edge tomorrow.

Reply:
Windows Chrome getBoundingClientRect values: `{height: 1275.3333740234375 , width: 1665.3333740234375}
Element clientHeight and ClientWidth values: `{height: 1275, width: 1665}`